operations/validate.go

```diff
@@ -29,13 +29,18 @@ func Validate() cli.Command {
 		}, cli.StringSliceFlag{
 			Name:  joinFlagNames(localModulesFlagName, "lm"),
 			Usage: "specify local modules as MODULE_NAME=PATH pairs",
+		}, cli.StringFlag{
+			Name:  joinFlagNames(projectFlagName, "p"),
+			Usage: "specify project identifier",
 		}),
 		Before: mergeBeforeFuncs(setPlainLogger, requirePathFlag),
 		Action: func(c *cli.Context) error {
 			confPath := c.Parent().String(confFlagName)
 			path := c.String(pathFlagName)
 			quiet := c.Bool(quietFlagName)
 			long := c.Bool(longFlagName)
+
+			projectID := c.String(projectFlagName)
```

Comment:
If projectID isn't specified, we can check and see if they have one in their local settings.  (this is the `projects_for_directory` section in your project evergreen.yml file) https://github.com/evergreen-ci/evergreen/blob/6158b431898d1492110cec3e9c5ac98508b5829d/operations/model.go#L216